components/infobox/wikis/mobilelegends/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,89 @@
+---
+-- @Liquipedia
+-- wiki=mobilelegends
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
```

Comment:
https://liquipedia.net/mobilelegends/index.php?title=Module:Earnings&action=edit
Earnings module should be in git before Infobox Player (as Infobox Player is dependent on the earnings module).